### ISSUE TITLE
Question about bundling axe-core

### ISSUE THREAD

**Issue by wcaneira** [component-driven/cypress-axe#37]
Thank you very much for contributing `cypress-axe` to the community!

I noticed that around `0.7.0` it was decided to bundle `axe-core` and I guess I'm wondering if would it might be better to keep `axe-core` as a peerDependency so that consumers can exert some level of control over their version?  I notice that there is already a request (#36) to upgrade the bundled version. 

Reasons why I ask:
1.  Maybe it's just a recent phenomenon, but I notice that Deque has had a few minor/patch releases to fine-tune the aggressiveness of the axe ruleset.  In my application I've had to upgrade a few times (to 3.4, to 3.5) to get around "overly aggressive" (my words, not theirs) errors that were not failing in other a11y checkers.  

1. I could be doing it wrong <sup>TM</sup> (and therefore it may not be a valuable use case), but in our application we currently also use `axe-core` directly in some of our faster JSDOM tests that are not covered by our slightly slower full-browser `cypress` tests.  While not maybe _technically_ necessary, it would be nice to have the a single `axe-core` version/ruleset apply.

Any thoughts are appreciated.  Thank you again for creating this useful library.

**Comment by avanslaars** [comment]
I bundled it to fix some issues with alternative build setups (webpack or TS instead of Cypress' default, browserify). I'm happy to revisit it, but I need to make sure it works for all the build configurations that were previously an issue for folks. I locked in at the version I did because axe-core was giving some false positives. I will look at #36 when I have some free time and at least work on getting the dependency updated.
